content/tutorials/use-tokens/create-an-automated-market-maker.md

```diff
@@ -0,0 +1,265 @@
+
+The helper function for issuing follows an abbreviated version of the steps in the [Issue a Fungible Token](issue-a-fungible-token.html) tutorial:
+
+<!-- MULTICODE_BLOCK_START -->
+
+_JavaScript_
+
+{{ include_code("_code-samples/create-amm/js/create-amm.js", language="js", start_with="/* Issue tokens") }}
+
+<!-- MULTICODE_BLOCK_END -->
+
+{{ start_step("Acquire tokens") }}
+<button id="buy-tst" class="btn btn-primary previous-steps-required">Buy TST</button>
+<button id="get-foo" class="btn btn-primary previous-steps-required">Get FOO</button>
+<div class="loader collapse"><img class="throbber" src="assets/img/xrp-loader-96.png">Working...</div>
+<div class="output-area"></div>
+{{ end_step() }}
+
+### {{n.next()}}. Check if the AMM exists
+
+Since there can only be one AMM for a specific pair of assets, it's best to check first before trying to create one. Use the [amm_info method][] to check whether the AMM already exists. For the request, you specify the two assets. The response should be an `actNotFound` error if the AMM does not exist.
+
+<!-- MULTICODE_BLOCK_START -->
+
+_JavaScript_
+
+{{ include_code("_code-samples/create-amm/js/create-amm.js", language="js", start_with="// Check if AMM already exists", end_before="// Look up AMM transaction cost") }}
+
+<!-- MULTICODE_BLOCK_END -->
+
+If the AMM does already exist, you should double-check that you specified the right pair of assets; if so, someone else has already created this AMM, but you can still deposit to it instead. <!-- TODO: link to a tutorial about depositing to and withdrawing from an AMM when one exists -->
+
+{{ start_step("Check for AMM") }}
+<button id="check-for-amm" class="btn btn-primary previous-steps-required">Check AMM</button>
+<div class="loader collapse"><img class="throbber" src="assets/img/xrp-loader-96.png">Sending...</div>
+<div class="output-area"></div>
+{{ end_step() }}
+
+### {{n.next()}}. Look up the AMMCreate transaction cost
+
+Creating an AMM has a special [transaction cost][] to prevent spam: since it creates objects in the ledger that no one owns, you must burn at least one [owner reserve increment](reserves.html) of XRP to send the AMMCreate transaction. The exact value can change due to [fee voting](https://xrpl.org/fee-voting.html), so you should look up the current incremental reserve value using the [server_state method][].
+
+It is also a good practice to display this value and give a human operator a chance to stop before you send the transaction. Burning an owner reserve is typically a much higher cost than sending a normal transaction, so you don't want it to be a surprise. (Currently, on both Mainnet and Devnet, the cost of sending a typical transaction is 0.000010 XRP but the cost of AMMCreate is 2 XRP.)
```

Comment:
I'd avoid using "Currently" to note the cost of AMMCreate as it could become outdated. Maybe "In September 2023"?

Reply:
We have several places in the docs that say something like this. I'd rather have it say, "Currently," and have to go update those when the numbers change, than to say, "As of September 2023," and have to constantly update it—if the date given is a year old, it may leave uncertainty about whether it's still accurate.

Transaction costs and reserves change rarely enough that I don't think it'll be a problem. The last change was in 2021 and before that they hadn't changed since before 2014. (They changed a couple times, I think, in the very early days, but I wasn't around for that.)